frontend/src/utils/theme-changer.ts

```diff
@@ -0,0 +1,10 @@
+import { useLayoutEffect } from 'react';
+
+const useThemeClass = () => {
+  useLayoutEffect(() => {
+    const storedTheme = localStorage.getItem('theme');
```

Comment:
so basically now we are inside the useLayoutEffect and outside we have useEffect ? two nesting is it preffered? 

useEffect -> useLayoutEffect . Can we not eliminate this, and have parent be useLayoutEffect in the app.tsx